src/app/team/shared/team-stage-select/team-stage-select.component.html

```diff
@@ -45,6 +45,9 @@
                </option>
             </select>
          </div>
+         <div *ngIf="competitionsBySeasonId[seasonIdFormValue]?.length === 0" class="d-flex align-items-center ml-2 mt-lg-2">
+            <div class="alert alert-warning mb-0 text-center" role="alert">Нажаль в цьому сезоні відсутні змагання</div>
```

Comment:
That's not an `alert-warning`, but rather `alert-info`. If possible, please use `<app-info>` component here.